src/background/settings.ts

```diff
@@ -3,24 +3,24 @@
  *
  * This contains only the settings that the background script uses. Other
  * settings are used when generating the `manifest.json` file.
- *
- * @typedef Settings
- * @prop {string} apiUrl
- * @prop {string} buildType
- * @prop {{ dsn: string, release: string }} [raven]
- * @prop {string} serviceUrl
- * @prop {boolean} [manifestV3]
  */
+export type Settings = {
+  apiUrl: string;
+  buildType: string;
+  raven?: { dsn: string; release: string };
+  serviceUrl: string;
+  manifestV3?: boolean;
+};
 
 // nb. This will error if the build has not been run yet.
 import settings from '../../build/settings.json';
 
 /**
  * Configuration data for the extension.
  */
-export default /** @type {Settings} */ ({
+export default {
   ...settings,
 
   // Ensure API url does not end with '/'
   apiUrl: settings.apiUrl.replace(/\/$/, ''),
-});
+} as Settings;
```

Comment:
Perhaps we could use `satisfies Settings` instead? Otherwise it will not complain even if the object does not actually fulfill `Settings`, which could lead to unintentional errors later on, due to typos and such.

```suggestion
} satisfies Settings;
```

Reply:
>  Otherwise it will not complain even if the object does not actually fulfill Settings

True. I modified this file so that the exported type is exactly `Settings`, but the conversion is done in a way that catches errors (`const settings: Settings = { ... }; export default settings;`).